gateway-core/components/org.wso2.carbon.gateway.core/src/main/java/org/wso2/carbon/gateway/core/flow/mediators/builtin/flowcontrollers/filter/Source.java

```diff
@@ -29,11 +31,24 @@
 
     private String value;
 
+    private PATHLANGUAGE pathLanguage;
```

Comment:
Its not auto generated. I wanted a generic name for XPath, JSONPath. And got the idea from [1]. I think better have it as PATH_LANGUAGE

[1] https://www.w3.org/TR/xpath/